integration_tests/tests/rpc/espace/test_debug_rpc.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+from integration_tests.test_framework.test_framework import ConfluxTestFramework
+from integration_tests.conflux.rpc import RpcClient
+from typing import Type
+from os.path import dirname, join
+from pathlib import Path
+import json
+from web3 import Web3
+toHex = Web3.to_hex
+
+# a temp util method, finally should be replaced by the util method provided by the framework
+# load contract metadata from tests/test_contracts/artifacts
+def _load_contract_metadata(name: str):
```

Comment:
Can try the function `load_contract_metadata` in `integration_tests/test_framework/utils/__init__.py` and see if it works